src/prefs/GUIPrefs.cpp

```diff
@@ -192,15 +192,7 @@ void GUIPrefs::PopulateOrExchange(ShuttleGui & S)
 #endif
    }
    S.EndStatic();
-
-   S.StartStatic(XO("Timeline"));
-   {
-      S.TieCheckBox(XXO("Show Scrub Ruler"),
-                    {wxT("/QuickPlay/ScrubbingEnabled"),
-                     false});
```

Comment:
The preference can also still be enabled with a button in the scrubbing toolbar.

OK, you are the designer, but I think it's too difficult to discover the scrubbing feature.  There is no longer, either, any clue about it in the context menu of the time ruler.

Scrubbing still needs a satisfactory user interface.

About your commit comment:  I follow the convention that if it doesn't all fit on one line, I end the first line with ellipsis ... and then there is one blank line before the rest.

Reply:
I did try to remove the entire scrubbing toolbar and scrub bar, however, doing so crashes Audacity upon startup as it reads the old preferences file. Any idea how to fix that?

Reply:
I could debug it if you wrote a draft PR, if you don't have the debugging experience.

But if you do all of that, how is anybody supposed to scrub at all?  How, besides using the Transport menu, awkwardly?